azure-pipelines.yml

```
@@ -69,14 +69,8 @@ stages:
             steps:
               - checkout: self
                 clean: true
-              - powershell: |
-                  SqlLocalDB stop
-                  SqlLocalDB delete
-                  Remove-Item "$env:USERPROFILE\*.mdf"
-                  Remove-Item "$env:USERPROFILE\*.ldf"
-                  SqlLocalDB create
-                  SqlLocalDB start
-                displayName: Cleanup databases
+              - script: tools\CleanMSSQLLocalDB.cmd
```

```suggestion
              - powershell: Get-ChildItem -Recurse -ErrorAction SilentlyContinue 'C:\Program Files*\sqlcmd.exe'
              - script: tools\CleanMSSQLLocalDB.cmd
```
Let's see if `sqlcmd.exe` is installed at all. If it is, can add directory to the path. If not, we can ask for that addition to the agent configuration.

To confirm: EFCore builds don't use `sqlcmd.exe` on the CI?